oap-server/server-starter/src/main/resources/ui-initialized-templates/activemq/activemq-destination.json

```diff
@@ -0,0 +1,286 @@
+[
+  {
+    "id": "ActiveMQ-Destination",
+    "configuration": {
+      "children": [
+        {
+          "x": 0,
+          "y": 0,
+          "w": 6,
+          "h": 11,
+          "i": "0",
+          "type": "Widget",
+          "id": "0",
+          "metricTypes": [],
+          "metrics": [],
+          "metricMode": "Expression",
+          "moved": false,
+          "expressions": [
+            "meter_activemq_destination_producer_count"
+          ],
+          "typesOfMQE": [
+            "TIME_SERIES_VALUES"
+          ],
```

Comment:
metricMode, metrics, metricTypes, typesOfMQE... have been removed from the latest UI, please sync UI and re-export template.

Reply:
ok

Reply:
Any changes about `table`

es monitoring panel about node health, not shown as expected

<img width="346" alt="image" src="https://github.com/apache/skywalking/assets/22817918/480fb999-1766-472f-a4c7-625eb21dc9b9">

Reply:
<img width="955" alt="image" src="https://github.com/apache/skywalking/assets/22817918/c74dde67-faa6-4175-8abf-45f06e9983d3">

Reply:
@Fine0830 Could you recheck this?

Reply:
Fixed by https://github.com/apache/skywalking-booster-ui/pull/389